x/gov/keeper/keeper.go

```diff
@@ -209,6 +225,25 @@ func (keeper Keeper) IterateInactiveProposalsQueue(ctx sdk.Context, endTime time
 	}
 }
 
+// IterateCanceledProposalQueue iterates over the proposal in the canceled proposal queue
+// and performs a callback function
+func (keeper Keeper) IterateCanceledProposalQueue(ctx sdk.Context, cb func(proposal v1.Proposal) (stop bool)) {
```

Comment:
Instead, can we just return an iterator function, e.g. like we do for `ActiveProposalQueueIterator` please?

Reply:
it is there https://github.com/cosmos/cosmos-sdk/pull/13010/files#diff-074187c6fc38ec544da75480f3a11a71d2855a635b5b478e9851be30b378c9b0R260